hypervisor/src/mshv/mod.rs

```diff
@@ -807,6 +814,17 @@ impl vm::Vm for MshvVm {
 
     /// Creates a guest physical memory region.
     fn create_user_memory_region(&self, user_memory_region: MemoryRegion) -> vm::Result<()> {
+        // We only keep the dirty log slots if it is not read only
+        if (user_memory_region.flags & HV_MAP_GPA_WRITABLE) != 0 {
```

Comment:
Looks to be a typo. Shouldn't it be `if (user_memory_region.flags & HV_MAP_GPA_WRITABLE) != HV_MAP_GPA_WRITABLE` ?